### PR TITLE
show multiple kapacitors

### PR DESCRIPTION
**Pull request by jademcgough**
- [ ] CHANGELOG.md updated
  - [x] Rebased/mergable
  - [x] Tests pass
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

Connect #1293 #1294 #1295 #1296 #1299, #1300

### The problem
Users currently cannot use multiple kapacitor configurations for a source.

### The Solution
- displays multiple kapacitor instances per source on the source management page
- provides a link to a "create kapacitor" page, where they can create a new kapacitor node
- allows users to edit existing kapacitor nodes
- persists the "active" kapacitor for a source, which is used on alerting pages

### CODE REVIEW

File: ui/src/sources/components/InfluxTable.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  return (
+    <Dropdown items={kapacitorItems} onChoose={() => {}} selected={kapacitorItems[0].text} />
+  )
+}
+
+const InfluxTable = ({
+  sources,
+  source,
+  handleDeleteSource,
+  location,
```
Comment (jademcgough) [inline ui/src/sources/components/InfluxTable.js]: I have nooo idea why I couldn't get `withRouter` to work with this component. When I wrapped the component with it, I wasn't getting `location` added to the props for some reason.

**Comment by jademcgough** [comment]
@cryptoquick says "I highly recommend making kapacitors its own reducer", which it probably the right call x.x

**Comment by alexpaxton** [comment]
Done styling!

**Comment by jademcgough** [comment]
@alexpaxton thank you!!! This looks amazing omg

File: bolt/servers.go [hunk 2]
```diff
@@ -118,3 +127,39 @@ func (s *ServersStore) Update(ctx context.Context, src chronograf.Server) error
 
 	return nil
 }
+
+func (s *ServersStore) all(ctx context.Context, tx *bolt.Tx) ([]chronograf.Server, error) {
```
Comment (goller) [inline bolt/servers.go]: Perhaps the `All` function could be gutted and use this new `all` func?

Comment (jademcgough) [inline bolt/servers.go]: 👍 good call

File: ui/src/sources/components/InfluxTable.js (hunk 1 above)
Comment (jaredscheib) [inline ui/src/sources/components/InfluxTable.js]: it should now work with the update to v3 of `react-router`, which was merged into master last night. our previous v2.4 did not support location props via `withRouter`.

Comment (jademcgough) [inline ui/src/sources/components/InfluxTable.js]: sweet, I need to rebase anyways

**Comment by jademcgough** [comment]
The nested state for kapacitors is a really bad idea here imo, though the one advantage was being able to map kapacitors to their sources easily. There should be reducer tests here as well.

**Pull request merged**